Feature frontend

nelsonfrank opened the pull request.

nelsonfrank committed:
Route Components using react-router-dom  #11

nelsonfrank committed:
Add FindRide and OfferRide Component  #12

BenMaruchu commented on frontend/src/Components/FindRide/Ride.js:
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Ride = ({ rideList }) => {
```
Why not map ride list inside `jsx` codes ?

BenMaruchu commented on frontend/src/Components/FindRide/RideList.js:
```diff
@@ -0,0 +1,24 @@
+    return (
+      <div>
+        <h2>Available Rides</h2>
+        <Ride rideList={this.state.RideOffer} />;
```
can you see how we did on todo-app on rendering list items

BenMaruchu commented on frontend/src/Components/FindRide/SearchBar.js:
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+
+class SearchBar extends Component {
```
Change this to stateless/ functional component

BenMaruchu reviewed (changes requested):
Hello @Nelsonfrank 

Nice work


I have seen you used couple of techniques in react which is nice but it is a best practice to pick one convention as use it across your codes i.e if you implement your components using arrow function make sure all you components are implemented that way if you choose functional way use that way across all component

nelsonfrank committed:
Add Find-ride component #12

nelsonfrank committed:
Add FindRide Component

FindRide Component Having Available Ride at a time and search functionality
for user to search the appropriate ride at a time #12

nelsonfrank committed:
Feat: Update App.js

nelsonfrank committed:
feat: Update SearchBar.js

nelsonfrank committed:
Feat: Add Search form to FindRide Component #12

nelsonfrank committed:
Feat: Add profile page

Profile Page containing Number of rides offered and number of rides taken #08

nelsonfrank committed:
Feat: Create OfferRide Component

The Component have input form for driver to provide information
about the ride #12

nelsonfrank committed:
Feat: Create Profile Component to display user information  #14

nelsonfrank committed:
Feat: Modify index.js

nelsonfrank committed:
Merge branch 'feature-frontend' of https://github.com/Nelsonfrank/Ride_My_Ways into feature-frontend
